ssm-run crashing due to missing waitgroup call

When trying to execute ssm-run targeting a sufficiently large number of instances, a missing call to [WaitGroup's Wait()](https://golang.org/pkg/sync/#WaitGroup.Wait) can cause the program to crash:

```go
ssm-run -c 'timeout 2 docker inspect $(docker ps -q)' -f '[TAGNAME]=[REDACTED]' -limit 400
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0x14e1364]
goroutine 19 [running]:
github.com/disneystreaming/go-ssmhelpers/ssm/invocation.GetCommandInvocationResult(0x17e2be0, 0xc0000a00b8, 0xc000096000, 0x190, 0x200, 0xc0002cda00, 0xc0003eac60, 0xc0003eacc0, 0xc000786300, 0x1)
	/home/runner/go/pkg/mod/github.com/disneystreaming/go-ssmhelpers@v0.1.0/ssm/invocation/runner.go:55 +0x224
github.com/disneystreaming/go-ssmhelpers/ssm.RunInvocations(0xc000099020, 0xc0000a00b8, 0xc000574000, 0x190, 0x400, 0xc0000a0070, 0x0, 0xc000099080, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/disneystreaming/go-ssmhelpers@v0.1.0/ssm/helpers.go:99 +0x3ac
main.main.func1(0xc0000a4ac0, 0xc000099040, 0xc0000a0070, 0xc000099020, 0xc000099080)
	/home/runner/work/ssm-helpers/ssm-helpers/cmd/ssm-run/main.go:216 +0x2cc
created by main.main
	/home/runner/work/ssm-helpers/ssm-helpers/cmd/ssm-run/main.go:190 +0xaf3
```

--- a/ssm/helpers.go
+++ b/ssm/helpers.go
@@ -95,6 +95,7 @@ func RunInvocations(sp *session.Pool, sess *ssm.SSM, instances []*ssm.InstanceIn
 		}
 	}
 
+	wg.Wait()
 	// Fetch the results of our invocation for all provided instances
 	invocationtatus, err := invocation.GetCommandInvocationResult(sess, commandOutput.Output...)
 	if err != nil {

